ipywidgets/widgets/widget_image.py

```diff
@@ -6,13 +6,22 @@
 Represents an image in the frontend using a widget.
 """
 
-import base64
-
 from .widget_core import CoreWidget
 from .domwidget import DOMWidget
 from .valuewidget import ValueWidget
 from .widget import register
-from traitlets import Unicode, CUnicode, Bytes, observe
+from traitlets import Unicode, CUnicode, Bytes, validate
+
+def _text_type():
```

Comment:
So this is a weird one. `setup.py` does not declare a dependency on `six`, but several of `ipywidgets`' dependencies do, including `traitlets`.

We can drop this whole `_text_type()` charade by just adding `six` to the dependency list, but I wasn't sure if that was something you wanted to do in a minor version, despite the fact that it will have no consequences whatsoever. You can change that in this PR or just do it in a second PR that replaces this `_text_type()` convenience function.

Reply:
I'm thinking that the next major version of ipywidgets will be python 3 only, and it's probably not worth it at this point to add a dependency on six for 4 lines of code.